pkg/dispatchcli/cmd/create.go

```diff
@@ -210,5 +224,6 @@ func NewCmdCreate(out io.Writer, errOut io.Writer) *cobra.Command {
 	cmd.AddCommand(NewCmdCreateEventDriverType(out, errOut))
 	cmd.AddCommand(NewCmdCreateApplication(out, errOut))
 	cmd.AddCommand(NewCmdCreatePolicy(out, errOut))
+	cmd.AddCommand(NewCmdCreateServiceAccount(out, errOut))
```

Comment:
Just talking with @neosab about the command naming.  I think we settled on having `iam` as a subcommand (i.e. `dispatch iam create serviceaccount`).  We are getting too many top-level resource types and we will also have serviceclass and serviceinstance resources which have nothing to do with iam.

Reply:
I will create iam subcommand and move policy & serviceaccount under it.